### ISSUE TITLE
SSL certificate error while running the sample code

### ISSUE THREAD

**Issue by nitinpurwar** [jessecambon/tidygeocoder#169]
> install.packages('tidygeocoder')
Retrieving 'https://cloud.r-project.org/src/contrib/tidygeocoder_1.0.5.tar.gz' ...
	OK [downloaded 199.5 Kb in 0.8 secs]
Installing tidygeocoder [1.0.5] ...
	OK [built from source]
Moving tidygeocoder [1.0.5] into the cache ...
	OK [moved to cache in 5.4 milliseconds]
> library(dplyr)
> library(tidygeocoder)
> dc_addresses <- tribble( ~name,~addr,
+                          "White House", "1600 Pennsylvania Ave Washington, DC",
+                          "National Academy of Sciences", "2101 Constitution Ave NW, Washington, DC 20418",
+                          "Department of Justice", "950 Pennsylvania Ave NW, Washington, DC 20530",
+                          "Supreme Court", "1 1st St NE, Washington, DC 20543",
+                          "Washington Monument", "2 15th St NW, Washington, DC 20024")
> coordinates <- dc_addresses %>%
+   geocode(addr)    
Passing 5 addresses to the Nominatim single address geocoder
_[-------------------------------------------------------------------------------------------------] 0/5 (  0%) Elapsed:  0s Remaining:  ?sError in curl::curl_fetch_memory(url, handle = handle) : 
  SSL certificate problem: certificate has expired_

**Comment by nitinpurwar** [comment]
which SSL cert is it looking for?

**Comment by nitinpurwar** [comment]
Hi @jessecambon,

The issue has been resolved. It was a root certificate issue for https://nominatim.openstreetmap.org/ui/search.html. 

I was getting this error in a controlled container environment. I had an image which had a expired root certificate (DST Root CA X3). To fix this issue, it was needed to have a latest cert  (ISRG Root X1) . I created an separate image with the this latest certificate. after importing this new customer, I was able to connect via SSL.

https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

Regards,
Nitin
